roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -266,8 +276,18 @@ spec:
 {% if rsyslog_command %}
           command: {{ rsyslog_command }}
 {% endif %}
+
 {% if rsyslog_args %}
           args: {{ rsyslog_args }}
+{% endif %}
+{% if rsyslog_host_network %}
+          hostNetwork: {{ rsyslog_host_network }}
+{% endif %}
+{% if rsyslog_dns_policy %}
+         dnsPolicy: {{ rsyslog_dns_policy }}
+{% endif %}
+{% if rsyslog_dns_policy == 'None' and rsyslog_dns_config is defined }}
```

Comment:
Similarly, there is an extra closing brace at the end of this line too which causes syntax errors when applied.